src/pages/help/help.html

```diff
@@ -36,13 +36,18 @@ <h1>{{APP_NAME}} Help</h1>
         </button>
 
         <p>
-            For each observation, please detail the type of observation, whether it is an observation of one 
-            or more animals or sign of the animal’s presence from the distinctive trail marks left on smooth barked eucalypts. 
+            For each observation, please detail the type of observation – either:
         </p>
 
-        <img src="assets/imgs/helpSlug.jpg" alt="A Giant Pink Slug" style="max-width: 100%">
-
-        <img src="assets/imgs/helpTrail.jpg" alt="A Giant Pink Slug's Trail" style="max-width: 100%">
+        <p>
+            an observation of one or more animals;
+            <img src="assets/imgs/helpSlug.jpg" alt="A Giant Pink Slug" style="max-width: 100%">
+        </p>
+        
+        <p>
+            or sign of the animal’s presence from the distinctive trail marks left on smooth barked eucalypts. 
```

Comment:
I've just learned the plural of eucalyptus, I was about that close to raise a typo comment.